Gemfile.lock

```diff
@@ -64,9 +64,9 @@ GEM
     acts_as_paranoid (0.4.3)
       activerecord (~> 3.2)
     acts_as_votable (0.7.1)
-    addressable (2.3.6)
+    addressable (2.3.8)

```

Comment:
@jsyeo Can we don't update all the gems or do this in another PR ? 

I'm afraid this could introduce some unknown bugs since we don't have enough test cases.  Like last time our html5 editor was broken after we installed a new version.